src/app/core/services/courseService/course.service.ts

```diff
@@ -29,9 +29,21 @@ export class CourseService {
 		return this.http.get(this.courseListUrl)
```

Comment:
These changes will allow us to get rid of `this.courseList` property. Because now we can use `getValue()` method of Behavior Subject to retrieve current data.